src/AutoRest.CSharp/Common/Output/Models/Shared/KnownParameters.cs

```diff
@@ -31,7 +31,7 @@ internal static class KnownParameters
 
         public static readonly Parameter RequestContext = new("context", "The request context, which can override default behaviors on the request on a per-call basis.", RequestContextNullableType, Constant.Default(RequestContextNullableType), false);
 
-        public static readonly Parameter WaitForCompletion = new("waitUntil", "Indicates whether this method should return once it has started the long-running operation or wait for the operation to fully complete before returning. For more information on long-running operations, please see <see href=\"https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/LongRunningOperations.md\"> Azure.Core Long-Running Operation samples</see>.", new CSharpType(typeof(WaitUntil)), null, false);
+        public static readonly Parameter WaitForCompletion = new("waitUntil", "\"F:WaitUntil.Completed\" if the method should wait to return until the long-running operation has completed on the service; \"F:WaitUntil.Started\" if it should return after starting the operation. For more information on long-running operations, please see <see href=\"https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/LongRunningOperations.md\"> Azure.Core Long-Running Operation samples</see>.", new CSharpType(typeof(WaitUntil)), null, false);
```

Comment:
Do we need to change this again? This will also impact the LLC side code

Reply:
@AlexanderSher make this change for llc last night, so both mgmt and llc will adopt the new lro change